setup.py

```diff
@@ -26,6 +26,7 @@ def get_readme():
 
 
 REQUIRED_PACKAGES = [
+    "conda-build",
```

Comment:
Does this requirement check installed conda packages?

Reply:
No, with pip, packages listed here would be checked for pip packages instead of conda one. So, if conda-build is already installed via conda, pip might not detect it and install pip package of conda-build too. I've not tried this although.

Reply:
Yeah, it feels like we shouldn't add this here. I'm not sure what advantage it gives us, and might cause some errors.

Reply:
Removed conda-build.